lightning_client.go

```diff
@@ -3085,6 +3086,33 @@ func (p *ChannelClosedUpdate) CloseTxid() chainhash.Hash {
 	return p.CloseTx
 }
 
+// CloseChannelOption is a functional type for an option that modifies a
+// CloseChannelRequest.
+type CloseChannelOption func(r *lnrpc.CloseChannelRequest)
+
+// SatPerVbyte is an option for setting the fee rate of a CloseChannelRequest.
+func SatPerVbyte(satPerVbyte uint64) CloseChannelOption {
```

Comment:
nit: shall we use `chainfee.SatPerVByte` here and for `MaxFeePerVByte`?

Reply:
fixed